File: j2k/tests/testData/ast/switch/fallDown.java
//file
package switch_demo;

public class SwitchDemo {
    public static void main(String[] args) {
        int month = 8;
        String monthString;
        switch (month) {
            case 1:
            case 2:
            case 3:
            case 4:
            case 5:
            case 6:
            case 7:
            case 8:
            case 9:
            case 10:
            case 11:
            case 12: monthString = "December"; break;
            default: monthString = "Invalid month"; break;
        }
        System.out.println(monthString);
    }
}

File: j2k/testData/fileOrElement/switch/fallDown.kt
package switch_demo

public class SwitchDemo {
    default object {
        public fun main(args: Array<String>) {
            val month = 8
            val monthString: String
            when (month) {
                1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12 -> monthString = "December"
                else -> monthString = "Invalid month"
            }
            System.out.println(monthString)
        }
    }
}

fun main(args: Array<String>) = SwitchDemo.main(args)